cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -49,6 +85,42 @@ describe("Tab management", () => {
       },
     };
 
+    describe("Changing workers shift type ", () => {
+      testWorkerData.hoursInfo = {
+        [HoursInfoCells.required]: 160,
+        [HoursInfoCells.actual]: 240,
+        [HoursInfoCells.overtime]: 80,
+      };
+      beforeEach(() => {
+        cy.get('[data-cy="btn-management-tab"]').click();
+        cy.get(`[data-cy="edit-worker-${testWorker}"]`).click();
+        cy.get('[data-cy="contract"]').click();
+      });
+      context(
+        "properly handles worker shift type from employment contract to, civil contract",
```

Comment:
Test się teraz czyta tak `"properly handles worker shift type from employment contract to, civil contract it properly handles worker shift type change"`, trochę dziwnie. Proponuję `context("when changing worker's shift type from employment contract to civil contract",... it("properly handles worker's shift type change",...`

Reply:
Zrobione